.travis/linux_apisix_master_luarocks_runner.sh

```diff
@@ -59,6 +59,9 @@ script() {
     sudo mkdir -p /usr/local/apisix/deps
     sudo PATH=$PATH ./utils/install-apisix.sh install > build.log 2>&1 || (cat build.log && exit 1)
 
+    which apisix
```

Comment:
You used the wrong way to rebase the branch.

This wrong result is usually due to the use of `merge` and `rebase` at the same time.

We have to close this PR first, then you can submit your correct code via new PR.

Reply:
ok, I will submit code via  new PR.

Reply:
the new pr: https://github.com/apache/apisix/pull/2592